Libraries/Text/RCTTextAttributes.m

```diff
@@ -237,6 +237,23 @@ - (UIColor *)effectiveBackgroundColor
   return effectiveBackgroundColor ?: [UIColor clearColor];
 }
 
+- (NSString *)capitalize:(NSString *)text
```

Comment:
can you make this a C function? i.e.

```
static NSString *CapitalizeText(NSString *text) {
  ...
}
```

this is more lightweight and the function does not require any state from this class.

Reply:
I think this is good. Tell me if it isn't!

As you can see with the code I wrote, I'm not familiar with Objective-C. I learned while I was making this PR 😁 So, thanks a lot for taking the time to review this PR, I learned a lot!